internal/build/imgsrc/docker.go

```diff
@@ -234,14 +254,30 @@ func newRemoteDockerClient(ctx context.Context, apiClient *api.Client, appName s
 			break
 		}
 	}
+
+	span.SetAttributes(
+		attribute.String("builder.name", remoteBuilderAppName),
+		attribute.String("builder.id", machine.ID),
+		attribute.String("builder.host", host),
+	)
+
 	if host == "" {
-		return nil, errors.New("machine did not have a private IP")
+		err = errors.New("machine did not have a private IP")
+		tracing.RecordError(span, err, "failed to boot remote builder")
+		return nil, err
 	}
 
 	builderHostOverride, ok := os.LookupEnv("FLY_RCHAB_OVERRIDE_HOST")
 	if ok {
 		oldHost := host
 		host = builderHostOverride
+
+		span.SetAttributes(
+			attribute.String("builder.old_host", oldHost),
```

Comment:
I think "old" is just the name of the previous value, not actually "old". Do we want to burn "old" into the attribute name?